packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-recipes.js

```diff
@@ -38,6 +38,53 @@ const FlowStageRecipes = {
    */
   initialiseSimpleC1C2BFlow(orderItemCriteriaList, logger, { c1ReqTemplateRef = null, c2ReqTemplateRef = null, bReqTemplateRef = null, brokerRole = null, taxMode = null } = {}) {
     // ## Initiate Flow Stages
+    const { fetchOpportunities, c1, c2, defaultFlowStageParams } = FlowStageRecipes.initialiseSimpleC1C2Flow(
+      orderItemCriteriaList,
+      logger,
+      {
+        c1ReqTemplateRef,
+        c2ReqTemplateRef,
+        brokerRole,
+        taxMode,
+      },
+    );
+    const b = new BFlowStage({
+      ...defaultFlowStageParams,
+      templateRef: bReqTemplateRef,
+      brokerRole,
+      prerequisite: c2,
+      getInput: () => ({
+        orderItems: fetchOpportunities.getOutput().orderItems,
+        totalPaymentDue: c2.getOutput().totalPaymentDue,
+        prepayment: c2.getOutput().prepayment,
+        positionOrderIntakeFormMap: c1.getOutput().positionOrderIntakeFormMap,
+      }),
+    });
+
+    return {
+      fetchOpportunities,
+      c1,
+      c2,
+      b,
+      // This is included in the result so that additional stages can be added using
+      // these params.
+      defaultFlowStageParams,
+    };
+  },
+  /**
+   * Initialise Flow Stages for a simple FetchOpportunities -> C1 -> C2 flow
+   *
+   * Rather than setting custom input for each stage, the input is just fed automatically
+   * from the output of previous stages.
+   *
+   * DO NOT USE THIS FUNCTION if you want to use custom inputs for each stage (e.g.
+   * to create erroneous requests).
+   *
+   * @param {OpportunityCriteria[]} orderItemCriteriaList
+   * @param {BaseLoggerType} logger
+   * @param {Omit<InitialiseSimpleC1C2BFlowOptions, 'bReqTemplateRef'>} [options]
+   */
+  initialiseSimpleC1C2Flow(orderItemCriteriaList, logger, { c1ReqTemplateRef = null, c2ReqTemplateRef = null, brokerRole = null, taxMode = null } = {}) {
```

Comment:
There's a lot of C1 -> C2 tests, so this was helpful

I was tempted to write an `initialiseSimpleC1Flow` as well but thought that might be pushing it